src/rules.jl

```diff
@@ -78,6 +78,33 @@ function (::typeof(frule_kwfunc))(kws::Any, ::typeof(frule), ::RuleConfig, args.
     return frule_kwfunc(kws, frule, args...)
 end
 
+struct ProductTangent{P}
+    partials::P
+end
+
+function frule(::RuleConfig{>:HasChunkedMode}, (Δf, Δx...), f, args...)
+    return frule((Δf, Δx...), f, args...)
+end
+
+function frule(::RuleConfig{>:HasChunkedMode},
+               (Δf, Δx)::Tuple{Any,ProductTangent}, f, args...)
+    fx = frule((Δf, first(Δx)), args...)[1]
+    dfx = ProductTangent(map(Δrow -> frule((Δf, Δrow), f, args...)[2], Δx))
+    return (fx, dfx)
```

Comment:
This calls the rule 3 times for 2 chunks.

Reply:
yeah. It's not ideal, but I wanted to throw something out there. Can you think of an easy way to remove the extra call while keeping the performance?

Reply:
Oh sorry, I was starting to make notes but didn't mean to click send... not really the top priority for a first cut!